### PR TITLE
[stable/openvpn] Update resource labels to latest best practices

### PR DESCRIPTION
**Pull request by wendorf**
**What this PR does / why we need it**:
- All resources directly created have `app`, `chart`, `release`, and `heritage`
- Because deployment pod selectors are immutable in API version apps/v1,
  openvpn deployment pods no longer have chart version in `chart` to
  allow updating.

### CODE REVIEW

**Comment by unguiculus** [comment]
You may also want the deployment to use `apps/v1beta2`.

**Comment by wendorf** [comment]
Good point! Just updated.

File: stable/openvpn/Chart.yaml
Comment (mattfarina) [inline stable/openvpn/Chart.yaml]: If you increment to version 3 it should be `3.0.0`. Can you fix that?

Comment (wendorf) [reply stable/openvpn/Chart.yaml]: Yep! Pushed a fix.

**Review by jfelten — APPROVED** [review]

**Comment by wendorf** [comment]
/hold

Trying to diagnose failures by pushing test commits.

**Comment by wendorf** [comment]
/hold cancel

Looks like prow won't run unapproved commits? What's the best way to diagnose this failure? I'm unable to reproduce in my own environment.

**Comment by wendorf** [comment]
/retest

From Slack (https://kubernetes.slack.com/archives/C6E3XH1ED/p1525364538000485):

> unguiculus [9:22 AM]
> *Charts CI had issues with PVCs not getting bound. I did some cluster maintenance today and things are working again.*

**Comment by wendorf** [comment]
@unguiculus Can you re-add the lgtm? I didn't understand the state machine for labels and pushed some test commits. They have been unpushed, but lgtm is now gone and tide won't run :(

**Pull request merged**
